### DISCUSSION TITLE
Is it normal for the database data to dissappear if there's too many concurrent calls to it / re-renders of the app?

### DISCUSSION THREAD

**Discussion by mpmua** [capacitor-community/sqlite#593] [Q&A]
I've noticed that when I accidentally trigger too many re-renders in React (In which case React complains and stops the app) and the database is called multiple times in quick succession, the data (or the actual database itself) will dissappear and on the next page refresh the database will be set up again from scratch with all previous data lost.

I was just wondering if this is normal or if I'm missing something here? Was just curious.

**Comment by jepiqueau** [comment]
@mpmua you must provide your code as i never experience this behavior
